src/main/java/org/gelecekbilimde/scienceplatform/post/model/request/CategoryCreateRequest.java

```diff
@@ -9,10 +12,27 @@
 @Setter
 @Builder
 public class CategoryCreateRequest {
-	@NotBlank
+
+	@NotBlank(message = "Category name cannot be blank")
+	@Size(max = 36, message = "Category name must be less than 36 characters")
 	private String name;
+
+	@NotBlank(message = "Description cannot be blank")
+	@Size(max = 255, message = "Description must be less than 255 characters")
+	private String description;
+
+	@NotNull(message = "Order number cannot be null")
+	@PositiveOrZero(message = "Order number must be zero or a positive integer")
 	private Integer order;
+
+	@NotBlank(message = "Slug cannot be blank")
+	@Size(max = 36, message = "Slug must be less than 36 characters")
 	private String slug;
+
+	@Size(max = 36, message = "Icon must be less than 36 characters")
```

Comment:
Özel bir mesaj yazmamaya çalışalımi buradaki mesajları bu şekilde özelleştirdiğimizde yazması zor olur ve uyuşmaz. Mümkün olduğunca spring'in kendi varsayılan mesajlarını kullanalım. 

@Ciklamert ikonları ön yüzde nasıl belirleyecez? Örnek bir komponent olup öyle mi seçilecek? Eğer bu şekildeyse oradaki ikon isimlerinin uzunluklarını bilyiorsak buna göre düzenlememizde fayda var.

```suggestion
	@Size(max = 36)
```